### PR TITLE
Committing changes for Foreign Key Oracle Tests

### PR DESCRIPTION
**Pull request by chetank-yb**

### CODE REVIEW

File: migtests/scripts/functions.sh [hunk 1]
```diff
@@ -54,6 +54,12 @@ run_mysql() {
 	mysql -u ${SOURCE_DB_USER} -p${SOURCE_DB_PASSWORD} -h ${SOURCE_DB_HOST} -P ${SOURCE_DB_PORT} -D ${db_name} -e "${sql}"
 }
 
+run_oracle() {
```
Comment (amit-yb) [inline migtests/scripts/functions.sh:57]: Name it as `run_sqlplus()`.

File: migtests/scripts/functions.sh [hunk 2]
```diff
@@ -54,6 +54,12 @@ run_mysql() {
...
+	db_name=$1
+	sql=$2
+	echo exit | sqlplus "${SOURCE_DB_USER}/${SOURCE_DB_PASSWORD}@(DESCRIPTION = (ADDRESS = (PROTOCOL = TCP)(HOST = ${SOURCE_DB_HOST}) (PORT = ${SOURCE_DB_PORT}))(CONNECT_DATA = (SID = ${db_name})))" @${sql}
```
Comment (amit-yb) [inline migtests/scripts/functions.sh:60]: Factor out the "connection string" into a separate variable named `conn_string` and use it in the cmd:
```
conn_string="${SOURCE_DB_USER}...."
echo exit | sqlplus ${conn_string} @${sql}
```

File: migtests/tests/ora-foreign-key-tests/foreign-key-schema.sql [hunk 3]
```diff
@@ -0,0 +1,172 @@
+-- regions
```
Comment (amit-yb) [inline migtests/tests/ora-foreign-key-tests/foreign-key-schema.sql:1]: Add comments in the schema explaining what SQL features are you validating.

File: migtests/tests/ora-foreign-key-tests/env.sh [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+export SOURCE_DB_USER="ADMIN"
+export SOURCE_DB_PASSWORD="Password1"
+export SOURCE_DB_HOST="cnaidu-oracle-rds.cbtcvpszcgdq.us-west-2.rds.amazonaws.com"
+export SOURCE_DB_SCHEMA="ADMIN"
+export TARGET_DB_HOST="10.9.200.131"
+export TARGET_DB_PORT="5433"
+export TARGET_DB_USER="ybvoyager"
+export TARGET_DB_PASSWORD="password"
+export TARGET_DB_NAME="migration_test"
+export TARGET_DB_SCHEMA="non_public_test"
```
Comment (amit-yb) [inline migtests/tests/ora-foreign-key-tests/env.sh:13]: This env.sh file is only meant to contain SOURCE_DB_TYPE, SOURCE_DB_NAME, and SOURCE_DB_SCHEMA.
Please, define the other variables in your shell environment.

File: migtests/scripts/functions.sh (hunk 2 above)
Comment (chetank-yb) [reply migtests/scripts/functions.sh:60]: fixed

File: migtests/scripts/functions.sh (hunk 1 above)
Comment (chetank-yb) [reply migtests/scripts/functions.sh:57]: Fixed

**Pull request closed without merging**